### ISSUE TITLE
Don't include perceval/__init.py__ when installing the pip package

### ISSUE THREAD

**Issue by acs** [chaoss/grimoirelab-perceval-mozilla#34]
When installing perceval-mozilla, if perceval/__init.py__ ins installed, it will overwrite the perceval main package version, breaking things.

The best approach is to not install this file when the perceval-mozilla package is installed.

Probably the same must be done also for: perceval/backend/__init.py__

I don't know is for doing that, the best option is just to remove perceval/__init.py__ because in python3 you don't need it. But maybe the current contents are needed for the perceval-mozilla development.

**Comment by jgbarah** [comment]
I'm having a look at this. From [Packaging namespace packages](https://packaging.python.org/guides/packaging-namespace-packages/):

> pkgutil-style namespace packages
>
> Python 2.3 introduced the pkgutil module and the extend_path function. This can be used to declare namespace packages that need to be compatible with both Python 2.3+ and Python 3. This is the recommended approach for the highest level of compatibility.
>
> To create a pkgutil-style namespace package, you need to provide an `__init__.py` file for the namespace package:
>
> The `__init__.py` file for the namespace package needs to contain only the following:
>
> `__path__ = __import__('pkgutil').extend_path(__path__, __name__)`
>
> Every distribution that uses the namespace package must include an identical `__init__.py`. If any distribution does not, it will cause the namespace logic to fail and the other sub-packages will not be importable. Any additional code in `__init__.py` will be inaccessible.

**Comment by jgbarah** [comment]
@acs can you describe a test to know if an installation is working properly or not? Something like "run this script and get this error".

**Comment by acs** [comment]
Yes:

In the working case:

```
acs@~ $ python3 -m venv /tmp/test-perceval
acs@~ $ source /tmp/test-perceval/bin/activate
acs@~ (test-perceval) $ pip3 install perceval
acs@~ (test-perceval) $ echo "from perceval import fetch" | python3 
acs@~ (test-perceval) $

```
After installing `perceval-mozilla` which breaks perceval (in some cases):

```
acs@~ (test-perceval) $ pip3 install perceval-mozilla
acs@~ (test-perceval) $ echo "from perceval import fetch" | python3 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'fetch'

```

**Comment by acs** [comment]
Let's test it with the latest version of the packages with changes to fix this issues:

```
acs@~ (venv) $ python3 -m venv /tmp/test-perceval
acs@~ (venv) $ source /tmp/test-perceval/bin/activate
acs@~ (test-perceval) $ pip3 install perceval
...
Successfully installed beautifulsoup4-4.6.0 certifi-2018.4.16 chardet-3.0.4 dulwich-0.18.6 feedparser-5.2.1 grimoirelab-toolkit-0.1.7 idna-2.6 perceval-0.10.3 python-dateutil-2.7.3 requests-2.18.4 six-1.11.0 urllib3-1.22
acs@~ (test-perceval) $ echo "from perceval import fetch" | python3 
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'fetch'
```

So in a first test, it seems that the issue is not fixed and now it is failing just installing perceval. @jgbarah am I doing something wrong?

**Comment by jgbarah** [comment]
There is no fetch in `perceval` now. In fact, `perceval` does not export any symbol now. If you need it, you need to install it from `perceval.backend`. Have a look at chaoss/grimoirelab-perceval#381

**Comment by vchrombie** [comment]
Addressed in #59

**Comment by vchrombie** [comment]
The future pip packages for this tool wouldn't have the `perceval/__init__.py` file and it is generated by poetry.
